chatterbox/src/Main/MainArea/Settings/user-settings-component.js

```diff
@@ -20,56 +22,115 @@ const SubmitChanges =  {
     color: '#FFFFFF'
 }
 
+const TextFieldStyles = {
+    width: '100%'
+}
+
 class UserSettings extends Component {
+    renderNormal() {
+        return (
+            <div>
+                <div className="user-name">
+                    <h2 className="user-detail">{this.props.user.user.name}</h2>
+                    <Pencil
+                        onClick={() => this.props.editUserName(this.props.user.user)}
+                        className="edit"
+                    />
+                </div>
+                
+                <div className="user-email">
+                    <h3 className="user-detail">{this.props.user.user.email}</h3>
+                    <Pencil
+                        onClick={() => this.props.editUserEmail(this.props.user.user)}
+                        className="edit"
+                    />
+                </div>
+            </div>
+        );
+    }
+
+    renderEditName() {
+        return (
+            <div>
+                <div className="user-name">
+                    <TextField  
+                        defaultValue={this.props.user.user.name} 

```

Comment:
user.user.name?

Reply:
Thank you for pointing that out. it has been fixed